lib/kubernetes-deploy/kubernetes_resource/memcached.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class Memcached < KubernetesResource
+    TIMEOUT = 5.minutes
```

Comment:
@devonboyer This is a very high timeout. Is it possible for this resource to take this long to come up under common circumstances?

Reply:
No that is probably to high. I just did the same thing as `redis`/`cloudsql` which also use seemingly crazy timeouts. I have an issue to refactor these TPR resources so I will add this to the list.